src/PageContent/Calendar/EventCard/EventCard.js

```diff
@@ -38,18 +39,26 @@ class EventCard extends Component {
         let iconClass = 'eventcard-icon ';
         let style = 'event-container ';
         let futureEvent = moment(delivery.startTimestamp).isAfter(moment());
+
         let actionNeeded = futureEvent && (
             (account.accountType === AccountType.DELIVERER_GROUP && !delivery.deliverers) ||
-            (account.accountType === AccountType.DONATING_AGENCY_MEMBER && 
+            (account.accountType === AccountType.DONATING_AGENCY_MEMBER &&
                 (!delivery.description || !delivery.description.foodItems)));
 
+        let emergencyDelivery = delivery.type === 'emergency';
+
         if (futureEvent) {
-            style += 'event-container-future';
+            //If it is an emergency delivery use .event-container-emergency-future class
+            emergencyDelivery ? style += 'event-container-emergency-future' : style += 'event-container-future';
             if (actionNeeded) {
                 // use red plus icon if an action is required for this delivery
                 icon = plus;
                 iconAlt = 'plus';
                 iconClass += 'eventcard-plus-icon';
+            } else if (emergencyDelivery) {
```

Comment:
Doing these conditionals is fine. Another approach could be similar to the utility mapping `src/PageLayout/Notification/NotificationMap.js`. We use it in the notifications to map `NotificationType` to things like what type of image to use, what the text should be, etc. You might consider making a similar mapping for calendar components. As we start to add different things to the calendar we might save ourselves a bit of code wrangling later down the road.

For now though this is probably fine, up to you!